storage/acl.go

```diff
@@ -259,6 +259,14 @@ func toBucketACLRules(items []*raw.BucketAccessControl) []ACLRule {
 	return rs
 }
 
+func toBucketACLRulesFromProto(items []*storagepb.BucketAccessControl) []ACLRule {
```

Comment:
I see some mix of `xxxFromProto`, `toXxxFromProto` and `fromProtoToXxx` in this file. Is there a reason for this or can we make them consistent?

Reply:
I've made everything conform to `toXxxFromProto` - basically the same as the apiary adpaters but with `FromProto` tacked on the end. Thanks for the comment.